R/teal_slice.R

```diff
@@ -234,6 +235,17 @@ filter_settings <- function(...,
 }
 
 
+# store -----------------------------------------------------------------------------------------------------------
+
+#' @param x object to be stored
+#' @param file storage file name
+#' @export
+#' @rdname teal_slice
+#' @keywords internal
+store <- function(x, file, ...) UseMethod("store")
```

Comment:
I'd be careful with name and rather use `store_filters` to avoid the conflicts with other packages.

Reply:
good suggestion @gogonzo

Reply:
https://github.com/insightsengineering/teal.slice/pull/339/commits/1c2d02d7a10825b7a2d0d77705d0e48413f1cbcf